.github/workflows/tests.yml

```diff
@@ -304,3 +304,113 @@ jobs:
           curl https://github.com/googleapis/repo-automation-bots/releases/download/flakybot-1.1.0/flakybot-darwin-amd64 -o flakybot -s -L
           chmod +x ./flakybot
           ./flakybot --repo ${{github.repository}} --commit_hash ${{github.sha}} --build_url https://github.com/${{github.repository}}/actions/runs/${{github.run_id}}
+
+  sample-tests:
```

Comment:
do we want these sample tests to run all the time? Or would it be better to have them only run when a sample file is updated?

Reply:
I think it wouldn't hurt to have them running all of the time, it's already down to a single target (ubuntu-latest on node18)